### ISSUE TITLE
LNK1112 Error

### ISSUE THREAD

**Issue by Teragrapher** [stievie/Urho3D_Template#1]
After trying to compile the project (Template.sln), I am getting an error:

LNK1112: module machine type 'X86' conflicts with target machine type 'x64'

I'm not exactly sure how to go about fixing this.

**Comment by stievie** [comment]
Have you been able to compile Urho3D before successfully?

Can you check these stepts please? http://stackoverflow.com/questions/3563756/fatal-error-lnk1112-module-machine-type-x64-conflicts-with-target-machine-typ

**Comment by Teragrapher** [comment]
It seems to have worked! I'll just list what I did in case someone else has the same issue:

1. Went to Build -> Configuration Manager
2. Changed Platform to Win32 (copying settings from x64 configuration)
3. Went to Project -> Properties -> Configuration Properties -> Linker -> Command Line, and removed the text from the Additional Options.

Unsure if this was the best way, but the link you sent sure did help.
